idph_etl/idph_etl.py

```diff
@@ -0,0 +1,269 @@
+import csv
```

Comment:
could you add a short description in https://github.com/uc-cdis/covid19-tools/blob/feat/idph_etl/README.md